core/src/main/java/hudson/BulkChange.java

```diff
@@ -113,6 +99,14 @@ public void commit() throws IOException {
     }
 
     /**
+     * Alias for {@link #abort()} to make {@link BulkChange} auto-closeable.
+     */
+    @Override
+    public void close() {
```

Comment:
:-1: abort will be invoked every time BulkChange is used as an auto-closable resource.